MediaBrowser.Model/Dlna/StreamBuilder.cs

```diff
@@ -1415,16 +1415,49 @@ private static int GetMaxAudioBitrateForTotalBitrate(long totalBitrate)
                 return profileMatch;
             }
 
+            TranscodeReason inferredReason = 0;
+
             var failureReasons = analyzedProfiles[false]
                 .Select(analysis => analysis.Result)
                 .Where(result => !containerSupported || (result.TranscodeReason & TranscodeReason.ContainerNotSupported) == 0)
```

Comment:
```suggestion
                .Where(result => !containerSupported || result.TranscodeReason.HasFlag(TranscodeReason.ContainerNotSupported))
```

Wasn't changed in this PR but this should also be `HasFlag` for readability